assets/js/myparcel.js

```diff
@@ -86,7 +86,6 @@ MyParcel = {
 
     setCurrentDeliveryOptions: function() {
         if (typeof MyParcel.storeDeliveryOptions === 'undefined') {
-            console.error('setCurrentDeliveryOptions() MyParcel.storeDeliveryOptions === undefined');
```

Comment:
Wat is de reden dat deze weg gehaald is?

Reply:
ja, inderdaad. Die console.error moet blijven

Reply:
Zie er het nut niet van in, misschien als de error daadwerkelijk zou zeggen wat er "mis" is zou het minder lijken op crappy code